do-catalog/src/styles/components/_buttons.scss

```diff
@@ -0,0 +1,112 @@
+.button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 10px 16px;
+  border-radius: 4px;
+  background: $button__bg-color;
+  color: $button__color;
+  font: 600 12px/16px "Montserrat", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
SCSS var for this font family definition.

Reply:
Updated!